### ISSUE TITLE
Transparent markers 

### ISSUE THREAD

**Issue by yakir12** [JuliaPlots/Plots.jl#1469]
I want to highlight something in an existing plot by circling it and annotating that. Something like this:
![bitmap](https://user-images.githubusercontent.com/3281957/38246239-6cb82198-3741-11e8-8d18-f4cd05c962c6.png)
I tried to just add a marker with a transparent face-color with this (under a `gr` backend):
```julia  
plot(rand(RGB, 10, 10))
scatter!(9*rand(10) + 1, leg=false, marker = (20, 0.0, stroke(3, 1.0, :black)))
```
But this doesn't work on the rendered window nor on the saved png.

**Comment by mkborregaard** [comment]
Yes, it looks like the markerstroke values override the marker values:
```julia
scatter(rand(10), markeralpha = 0.2, ms = 20)
```
![skaermbillede 2018-04-03 kl 13 53 50](https://user-images.githubusercontent.com/8429802/38247732-914b55c0-3746-11e8-8178-de3d1a09716d.png)

```julia
scatter(rand(10), markeralpha = 0.2, ms = 20, markerstrokealpha = 1)
```
![skaermbillede 2018-04-03 kl 13 54 30](https://user-images.githubusercontent.com/8429802/38247739-957f06b4-3746-11e8-8994-eedbcf407d98.png)

**Comment by yakir12** [comment]
Just a hopefully helpful note to anyone stumbling here, using an `RGBA` works for the annotations:
```julia
annotate!(x, y, text("half transparent red label", RGBA(1,0,0, 0.5)))
```
![a](https://user-images.githubusercontent.com/3281957/38353119-c6b4d98c-38b5-11e8-9feb-0ca07a828328.png)

**Comment by mkborregaard** [comment]
Ugh, that's unfortunate. Replacing markers with Shapes is likely to really slow down plotting.

**Comment by mkborregaard** [comment]
Of course @yakir12 's original use case can be solved by plotting a circle shape. (e.g. `plot(Plots.partialcircle(0, 2π, 100))` if the aspect ratio is 1).
